src/main/java/com/stripe/param/issuing/CardCreateParams.java

```diff
@@ -3392,7 +3389,7 @@ public Builder setType(Type type) {
       }
 
       /** Packaging options. */
-      public Builder setType(EmptyParam type) {
+      public Builder setType(String type) {
```

Comment:
I don't fully grasp this change. We had an enum already, why now `Object` and also why did `setType` lose the `EmptyParam` one?

Reply:
I also think this is breaking

Reply:
Not entirely sure why we had the `EmptyParam` overload in the first place, this was probably a side-effect of one of the hacks we added to the Java codegen to handle unsettable fields.

The change is likely because the parameter was converted from an old-style enum (`StringParam` with a set of `values`) to a new-style enum (`APIEnumParam`).

This is technically breaking, but I don't think it matters since sending an empty value in the creation request would have failed anyway.

Reply:
Actually this seems to be fixed already in the latest master. Can you try rebasing?